pkg/admission/webhook/generic_webhook.go

```diff
@@ -160,11 +149,8 @@ func (p *WebhookDispatcher) getAPIExportCluster(attr admission.Attributes, clust
 	for _, apiBinding := range objs {
 		for _, br := range apiBinding.Status.BoundResources {
 			if br.Group == attr.GetResource().Group && br.Resource == attr.GetResource().Resource {
-				path := logicalcluster.NewPath(apiBinding.Spec.Reference.Export.Path)
-				if path.Empty() {
-					path = clusterName.Path()
-				}
-				export, err := p.getAPIExport(path, apiBinding.Spec.Reference.Export.Name)
+				name, _ := logicalcluster.NewPath(apiBinding.Status.LogicalCluster).Name()
```

Comment:
Don't need to create a path and then convert back to a name - you can do clusterName := logicalcluster.Name(apiBinding.Status.LogicalCluster)